src/objectExplorer/objectExplorerProvider.ts

```diff
@@ -0,0 +1,161 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as vscode from 'vscode';
+import ConnectionManager from '../controllers/connectionManager';
+import { NodeInfo } from '../models/contracts/objectExplorer/nodeInfo';
+import { ObjectExplorerService } from './objectExplorerService';
+import { ConnectionCredentials } from '../models/connectionCredentials';
+
+export class TreeNodeInfo extends vscode.TreeItem {
```

Comment:
>TreeNodeInfo [](start = 13, length = 12)

I'd suggest putting this into its own file - no reason to overload this file with additional stuff